ext/rbsecp256k1/rbsecp256k1.c

```diff
@@ -1102,36 +1102,57 @@ Context_alloc(VALUE klass)
  *
  * Context initialization should be infrequent as it is an expensive operation.
  *
+ * @param in_random_bytes [String,nil] (Optional) 32 bytes of random data used
+ *   to randomize the context. If omitted then the context remains
+ *   unrandomized. It is recommended that you provide this argument.
  * @return [Secp256k1::Context] 
  * @raise [RuntimeError] if context randomization fails.
  */
 static VALUE
-Context_initialize(VALUE self)
+Context_initialize(int argc, const VALUE* argv, VALUE self)
 {
   Context *context;
   unsigned char *seed32;
   VALUE random_bytes;
+  VALUE opts;
+  static ID kwarg_ids[1];
```

Comment:
It is probably possible to get away with not even making this an array since there is only one element. This would simplify the initialization code below.